src/app/shared/services/chat/chat.service.ts

```diff
@@ -11,18 +11,20 @@ export class ChatService {
   constructor(private http: HttpClient) {}
 
   getChatRooms(role: Role): Observable<ChatRoom[]> {
-    return this.http.get<ChatRoom[]>('assets/mocks/chatrooms.json');
-    //TODO: return this.http.get<ChatRoom[]>(`/api/v1/ChatWorkshop/${role}/chatrooms`);
+    return this.http.get<ChatRoom[]>(`/api/v1/ChatWorkshop/${role}/chatrooms`);
   }
 
-  getChatRoomsMessages(
-    chatRoomId: string,
-    parameters: MessagesParameters
-  ): Observable<Message[]> {
-    return this.http.get<Message[]>('assets/mocks/messages.json');
+  //TODO: reduce the number of arguments
+  getChatRoomsMessages(chatRoomId: string, role: Role, parameters: MessagesParameters): Observable<Message[]> {
+    let params = new HttpParams();
```

Comment:
Just a small remark,
you can use chaining here, like:
`params = new HttpParams()
    .set('page', '2')
    .set('page', '3')
    .set('sort', 'name');`